### ISSUE TITLE
Creatures should be saved in one line in level maps

### ISSUE THREAD

**Issue by hwoarangmy** [OpenDungeons/OpenDungeons#520]
ATM, in the level files, creatures are defined between [Creature] tags. That allows to have the creature stats as we want, within several lines if needed. But since there can be many creatures per file, I would propose to remove the tags and to have the rule that a creature must be defined in one line.
I know I was the one to introduce the Creature tags but after all, it may not be a good idea ^^

**Comment by akien-mga** [comment]
Would that be to prepare the ground for the load/save feature?

Just to make it clearer, do you speak about creature instancing (`[Creatures]`) or creature (classes) definitions (`[CreatureDefinitions]`)? IMO for the latter, the `[Creature]` tags should be kept, and settings should be kept each on its own line for legibility matters.

For the creature instancing, the `[Creature]` tags are indeed redundant and can safely be removed IMO, as in:

```
[Creature]
0 BossDragon1 BossDragon 62 14 0 BossDragon 4 0 max 100 0 0 none none
[/Creature]
[Creature]
0 BossDragon2 BossDragon 40 68 0 BossDragon 3 0 max 100 0 0 none none
[/Creature]
```

**Comment by hwoarangmy** [comment]
> Would that be to prepare the ground for the load/save feature?

Who knows ? ;-)

> do you speak about creature instancing

yes, like in your example. For creature definitions, it makes much more sense to use tags.

**Comment by Bertram25** [comment]
Fine to me, as long as we're using one line and Creatures start and ending tags. :)

**Comment by hwoarangmy** [comment]
ok, so let's do that ^^

**Comment by hwoarangmy** [comment]
fixed in #527

**Comment by Bertram25** [comment]
Hi there, just a rush note wildly placed at a random location.
Sorry, I'm very busy and quite out of network those days and I won't be available until monday afternoon at least.
Sorry to let you alone guys but I'm sure you're doing fine. ;)

Regards,
